src/main/java/practice/CandidateValidator.java

```diff
@@ -1,5 +1,38 @@
 package practice;
 
-public class CandidateValidator {
-    //write your code here
+import java.util.function.Predicate;
+import model.Candidate;
+
+public class CandidateValidator implements Predicate<Candidate> {
+    private static final int REQUIRED_CANDIDATE_AGE = 35;
```

Comment:
```suggestion
    private static final int MIN_CANDIDATE_AGE = 35;
```

since it should be **at least** 35 years old, not exactly